website/routes.py

```diff
@@ -183,8 +186,11 @@ def make_url_map(app):
 
         Rule('/about/', 'get', website_views.redirect_about, json_renderer,),
         Rule('/howosfworks/', 'get', website_views.redirect_howosfworks, json_renderer,),
-        Rule('/faq/', 'get', {}, OsfWebRenderer('public/pages/faq.mako')),
-        Rule('/getting-started/', 'get', {}, OsfWebRenderer('public/pages/getting_started.mako')),
+
+        Rule('/faq/', 'get', lambda *args, **kwargs: redirect('/support/'), OsfWebRenderer('public/pages/support.mako')),
```

Comment:
I was fixing this with Jeff, we did it this way because of a quirk with flask I think, it wouldn't allow me use the same function in both of these